packages/core/src/components/badge/badge.tsx

```diff
@@ -15,6 +15,11 @@ export class TdsBadge {
   /** Sets component size. */
   @Prop() size: 'lg' | 'sm' = 'lg';
 
+  /** Defines aria-live attribute */
+  @Prop() ariaLiveAnnounce: 'off' | 'polite' | 'assertive' = 'polite';
+
+  @State() ariaLabelValue: string = '';
+
```

Comment:
```suggestion
  /** Defines aria-live attribute */
  @Prop() tdsAriaLive: 'off' | 'polite' | 'assertive' = 'polite';

  @State() ariaLabelValue: string = '';

```

Reply:
`ariaLabelValue` should probably also be changed. `tdsAriaLabel`

Reply:
I this prefixing all of these with `tds` works fine. Could even add a page in storybook to document this

Reply:
ariaLabelValue as it is state could remain as it wont create issue, but for the sake of convention we should probably do that one too.